modules/oe_subscriptions_anonymous/src/AnonymousSubscriptionManager.php

```diff
@@ -22,7 +22,10 @@ class AnonymousSubscriptionManager implements AnonymousSubscriptionManagerInterf
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
    *   The entity type manager.
    */
-  public function __construct(protected FlagServiceInterface $flagService, protected EntityTypeManagerInterface $entityTypeManager) {}
+  public function __construct(
+    protected readonly FlagServiceInterface $flagService,
+    protected readonly EntityTypeManagerInterface $entityTypeManager,
+  ) {}
```

Comment:
As a general note:
The `readonly` can cause problem in combination with `DependencySerializationTrait`, this is why we cannot use it on form classes.
But in most other places it is fine.